addons/popochiu/engine/objects/room/popochiu_room.gd

```diff
@@ -405,6 +405,12 @@ func set_active_walkable_area(walkable_area_name: String) -> void:
 		PopochiuUtils.print_error("Can't set %s as active walkable area" % walkable_area_name)
 
 
+func get_move_target_position(start_position: Vector2, end_position: Vector2) -> Vector2:
```

Comment:
Is this a public utility function? Does it need to be documented (API wise?)

Reply:
Yes. I forgot to document it. Thanks!